src/sema/types.rs

```diff
@@ -1052,8 +1054,9 @@ impl Type {
                 ty.solana_storage_size(ns).mul(
                     dims.iter()
                         .map(|d| match d {
-                            None => &pointer_size,
-                            Some(d) => d,
+                            ArrayLength::Dynamic => &pointer_size,
+                            ArrayLength::Fixed(d) => d,
+                            ArrayLength::AnyFixed => panic!("unknown size"),
```

Comment:
```suggestion
                            ArrayLength::AnyFixed => panic!("unknown length"),
```
You used `unknown length` down below.

Reply:
Fixed, thanks